utils/build.config

```diff
@@ -4,5 +4,5 @@ component=cart
 [commit_versions]
 OFI = 955f3a07dd011fb1dbfa6b6c772ada03d5af320e
 OPENPA = v1.0.4
-MERCURY = 9889a0d0ba8854c1a0c2bd186a01fac4f57edada
+MERCURY =c2c262813811c3ede28ee32fdebbffd417a7cb80
```

Comment:
Not sure if you care, but you removed the whitespace here:

```suggestion
MERCURY = c2c262813811c3ede28ee32fdebbffd417a7cb80
```

Reply:
good catch. I will add back the space.